app/views/checkEligibility/InEligibleCharitablePurposesView.scala.html

```diff
@@ -27,14 +27,14 @@
 
 @()(implicit request: Request[_], messages: Messages, appConfig: FrontendAppConfig)
 
-@govukLayout(pageTitle = Some(titleNoForm("inEligibleCharitablePurposes.title", None)), signOut = false) {
+@govukLayout(pageTitle = Some(titleNoForm("notEligible.title", None)), signOut = false) {
 
-    @h1(messages("inEligibleCharitablePurposes.heading"))
+    @h1(messages("notEligible.heading"))
 
-    @p(Html(messages("inEligibleCharitablePurposes.p1")))
-    @p(Html(messages("inEligibleCharitablePurposes.p2")))
+    @p(Html(messages("notEligible.charitablePurposes.p1")))
+    @p(Html(messages("notEligible.p2")))
     @p{
-        @messages("inEligibleCharitablePurposes.p3")
-        @link("https://www.gov.uk/charities-and-tax/get-recognition",messages("inEligibleCharitablePurposes.p3.link"), isExternal = true)
+        @messages("notEligible.p3")
+        @link("https://www.gov.uk/charities-and-tax/get-recognition",messages("notEligible.p3.link"), isExternal = true)
```

Comment:
can we move this link to application.conf and then read it using appConfig please? its applicable for other inEligible pages